### DISCUSSION TITLE
How long will podman version 4 be supported and maintained?

### DISCUSSION THREAD

**Discussion by bitcindy** [containers/podman#23090] [General]
How long will podman version 4 be supported and maintained?

**Comment by Luap99** [comment]
Depends how you define supported. We only support the latest release upstream generally.

Podman 4.9 however will still get security fixes if necessary as it is still in fedora 39 but only until f39 is EOL.

**Reply by rhatdan** [reply]
Correct we only support latest and many of the upstream developers support Fedora, so since Fedora 39 can not be updated to 5, because of Fedora rules on no breaking change, we support it until some time in November.

Red hat pays some of the upstream developers to back port fixes for RHEL versions of Podman.  Do not know if SUSE does similar things.

**Reply by bitcindy** [reply]
Got it. Thank you so much!
